src/bucket/upload_part.gleam

```diff
@@ -0,0 +1,64 @@
+import bucket.{type BucketError, type Credentials}
+import bucket/internal
+import gleam/http
+import gleam/http/request.{type Request, Request}
+import gleam/http/response.{type Response}
+import gleam/int
+import gleam/list
+import gleam/option
+import gleam/result
+
+pub type UploadPartResult {
+  UploadPartResult(etag: String)
+}
+
+/// The parameters for the API request
+pub type RequestBuilder {
+  RequestBuilder(
+    bucket: String,
+    key: String,
+    upload_id: String,
+    part_number: Int,
```

Comment:
What is a part number?

Reply:
Document this please 🙏

Reply:
Could you add this a doc comment please 🙏

Reply:
done in fd7e3c6